### ISSUE TITLE
Is it possible to use the new Inbox interface to Gmail?

### ISSUE THREAD

**Issue by damag** [davesteele/gnome-gmail#7]
Hi, thanks for making this app. I was wondering if you had any plans to support the new [Inbox interface](https://inbox.google.com/)?

**Comment by davesteele** [comment]
Not likely. There would need to be a reference on how to craft a URL to open an existing Draft. I don't see that.

**Comment by damag** [comment]
Good point, maybe they'll provide that someday. Composing via the old interface isn't a big deal anyways. Thanks for looking into it.

**Comment by Queuecumber** [comment]
Hi @davesteele 
its 

`https://inbox.google.com/u/0/?mailto=mailto:<email address here>`

No official docs but you can verify this by allowing chrome to open mailto links using inbox and grabbing the URL. Note that it will try to hide this URL from you but I got it using the back button.

**Comment by davesteele** [comment]
Thanks. I'll give it a try.

**Comment by ddnexus** [comment]
Any update?

**Comment by davesteele** [comment]
I don't see how to reliably specify the account using that url.

**Comment by ddnexus** [comment]
What about just generating the URL from a configurable string in the
settings? Something  like a pattern with dynamic parts that would be filled
by default with the gmail pattern, but the user could edit in order to
experiment with inbox too?

That way would overcome the general reliability that an automatic
generation would require, supplying a local way to customize it.

**Comment by davesteele** [comment]
Too much of a kluge to formalize as a configuration parameter. If you want to go it alone, the URLs are generated in the *_gmail_url() methods in gnomegmail.py.

**Comment by ddnexus** [comment]
That's good to know. Thank you!
